src/app/modules/group/components/associated-activity/associated-activity.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, forwardRef, OnDestroy } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { Router } from '@angular/router';
+import { merge, of } from 'rxjs';
+import { Subject } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { GetItemByIdService } from 'src/app/modules/item/http-services/get-item-by-id.service';
+import { fetchingState, isReady, readyState } from 'src/app/shared/helpers/state';
+
+@Component({
+  selector: 'alg-associated-activity',
+  templateUrl: './associated-activity.component.html',
+  styleUrls: [ './associated-activity.component.scss' ],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => AssociatedActivityComponent),
+      multi: true,
+    }
+  ]
+})
```

Comment:
Great I didn't know that there was this alternative for handling form component. Actually it may apply to many things done by Amaury.